packages/desktopjs-electron/src/electron.ts

```diff
@@ -231,10 +230,10 @@ export class ElectronContainerWindow extends ContainerWindow {
  * @augments MessageBus
  */
 export class ElectronMessageBus implements MessageBus {
-    public ipc: NodeJS.EventEmitter;
+    public ipc: any;
```

Comment:
Getting rid of exposed node types to avoid upstream pain when devs attempt to build.

Reply:
we should mention it in release notes as a breaking change

Reply:
Will be in release notes